api/apiFunctions.test.tsx

```diff
@@ -0,0 +1,43 @@
+ import { loginAccount, logoutAccount } from './apiFunctions';
+import { account } from './config';
+
+describe('Auth Functions', () => {
+
+  jest.mock('./apiFunctions', () => ({
+    loginAccount: jest.fn()
+  }))
+
+  describe('login account successful', () => {
+    it('should show user login successfully', async () => {
+      const userDummy = { 
+          email: "testemail@email.com",
+          password: "test1234",
+        }
+     await loginAccount(userDummy);
+      expect(account.createEmailPasswordSession).toBeInstanceOf(Object);
```

Comment:
Is there a reason why you mocked loginAccount, but you didn't mock createEmailPasswordSession.

Reply:
I thought loginAccount was the same as createEmailPasswordSession, therefore, I didn't want to duplicate work.